.wordlist.txt

```diff
@@ -1,18 +1,40 @@
+addons
+API
+APIs
+balancer
+CharmHub
+CLI
+dropdown
 Diátaxis
+EBS
+EKS
 favicon
-https
+filesystem
+Grafana
+hostname
+IAM
 installable
+JSON
+Juju
+Kubernetes
+Kubeflow
+lifecycle
 Makefile
 MyST
+namespace
+namespaces
+NodePort
+observability
+OLM
 Permalink
 ReadMe
+reproducibility
```

Comment:
Where do we need this?!
"Can be reproduced" is a MUCH easier way of saying the same thing. ;)